### DISCUSSION TITLE
Self nominate Amelia Castilla (@accrazed) as a NodeJS Contributor

### DISCUSSION THREAD

**Discussion by accrazed** [paketo-buildpacks/feedback#127] [Node.js Team]
In accordance with the [Paketo Buildpacks Governance document](https://github.com/paketo-buildpacks/community/blob/main/GOVERNANCE.md), I am nominating myself, Amelia Castilla (@accrazed), as a contributor to the NodeJS language family. I have contributed to the buildpack family:
- https://github.com/paketo-buildpacks/node-run-script/pull/1
- https://github.com/paketo-buildpacks/nodejs/pull/417

**Comment by arjun024** [comment]
I vote yes!

**Comment by thitch97** [comment]
I vote yes

**Comment by ForestEckhardt** [comment]
@paketo-buildpacks/steering-committee I believe that this is a super majority of Nodejs maintainers approval
